test/unit/mock/cmd/firewall-cmd--get-active-zones

```diff
@@ -0,0 +1,2 @@
+public
```

Comment:
I'd like to see additional zones added to this so we're sure `parse_active_zones` operates properly. You'll then need to modify your tests given that you'll have more than one zone to work with.